python/ray/data/_internal/stats.py

```diff
@@ -227,8 +227,16 @@ def TODO():
         """Placeholder for ops not yet instrumented."""
         return DatasetStats(stages={"TODO": []}, parent=None)
 
-    def summary_string(self, already_printed: Set[str] = None) -> str:
-        """Return a human-readable summary of this Dataset's stats."""
+    def summary_string(
+        self, already_printed: Set[str] = None, include_parent: bool = True
+    ) -> str:
+        """Return a human-readable summary of this Dataset's stats.
+
+        Args:
+        `already_printed`: Set of stage IDs that have already had its stats printed out.
```

Comment:
The ticks aren't necessary for these args to be rendered as code/monospaced, that is already taken care of by RST/Sphinx.
```suggestion
        already_printed: Set of stage IDs that have already had its stats printed out.
```